components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/test/java/org/wso2/carbon/identity/application/authentication/framework/util/FrameworkUtilsTest.java

```diff
@@ -88,11 +130,160 @@ public void testGetAppAuthenticatorByNameValidAuthenticator(){
     }
 
     @Test
-    public void testGetAppAuthenticatorByNameNonExistAuthenticator(){
+    public void testGetAppAuthenticatorByNameNonExistAuthenticator() {
         ApplicationAuthenticator out = FrameworkUtils.getAppAuthenticatorByName("nonExistingAuthenticator");
         assertNull(out);
     }
 
+    @DataProvider(name = "provideRequestCoordinators")
+    public Object[][] provideRequestCoordinators() {
+
+        Map<String, Object> map1 = new HashMap<>();
+        map1.put(FrameworkConstants.Config.QNAME_EXT_REQ_COORDINATOR, testRequestCoordinator);
+        return new Object[][]{
+                {map1, testRequestCoordinator}
+        };
+    }
+
+    @Test(dataProvider = "provideRequestCoordinators")
+    public void testGetStepBasedSequenceHandlerExistReqCoordinator(Map<String, Object> instance, Object expectedOutput) {
```

Comment:
Add new line after the method signature

Reply:
Fix in the other places too..

Reply:
Already fixed with commit baf49d4b0e85432467ae5848dcf443c0032ce5c9 in https://github.com/wso2/carbon-identity-framework/pull/3509 PR